### PR TITLE
Update MySQL credential setting locations

### LINKED ISSUE #322: Problem: MySQL security section is obsolete
**Version of the documentation**

1.8

**Page (and section, if applicable) where the issue occurs**

https://github.com/artefactual/archivematica-docs/blob/1.8/admin-manual/security/security.rst#mysql

**Description of the issue**

The documentation uses old files to change MySQL credentials:

<pre>
/etc/archivematica/archivematicaCommon/dbsettings
/usr/share/archivematica/dashboard/settings/common.py 
</pre>

Those files doesn't exist for AM18

**Suggested fix**

This section should use the `/etc/default/archivematica-*` files for Ubuntu and `/etc/sysconfig/archivematica-*` files for CentOS instead.

### PR DESCRIPTION
**Pull request by ablwr**
Resolve https://github.com/archivematica/Issues/issues/322

### CODE REVIEW

File: admin-manual/security/security.rst [hunk 1]
```diff
@@ -50,16 +50,22 @@ the following into the command-line:
       FLUSH PRIVILEGES;"
 
 Once you've done this you can change Archivematica's MySQL database access
-credentials by editing these two files:
+credentials by editing your system files and changing the `CLIENT_USER` and
```
Comment (mamedin) [inline admin-manual/security/security.rst]: Hi, the variable names are a bit different in each AM service.

Dashboard: 
- ARCHIVEMATICA_DASHBOARD_CLIENT_DATABASE
- ARCHIVEMATICA_DASHBOARD_CLIENT_HOST
- ARCHIVEMATICA_DASHBOARD_CLIENT_PASSWORD
- ARCHIVEMATICA_DASHBOARD_CLIENT_USER

MCPClient:
- ARCHIVEMATICA_MCPCLIENT_CLIENT_DATABASE
- ARCHIVEMATICA_MCPCLIENT_CLIENT_HOST
- ARCHIVEMATICA_MCPCLIENT_CLIENT_PASSWORD
- ARCHIVEMATICA_MCPCLIENT_CLIENT_USER

MCPServer:
- ARCHIVEMATICA_MCPSERVER_CLIENT_DATABASE
- ARCHIVEMATICA_MCPSERVER_CLIENT_HOST
- ARCHIVEMATICA_MCPSERVER_CLIENT_PASSWORD
- ARCHIVEMATICA_MCPSERVER_CLIENT_USER

Storage Service when using sqlite3:
- SS_DB_NAME
- SS_DB_HOST
- SS_DB_PASSWORD
- SS_DB_USER

Storage Service when using MySQL:
- SS_DB_URL  

For SS MySQL database see: [SS application-specific-environment-variables ](https://github.com/artefactual/archivematica-storage-service/blob/stable/0.15.x/install/README.md#application-specific-environment-variables) :

"The configuration of the database is also declared via environment variables. Storage Service looks up the SS_DB_URL environment string. If defined, its value is expected to follow the form described in the dj-database-url docs, e.g.: mysql://username:password@192.168.1.20:3306/storage_service. If undefined, Storage Service defaults to the django.db.backends.sqlite3 engine and expects the following environment variables to be defined"

**Review by mamedin — CHANGES_REQUESTED** [review]

**Comment by ablwr** [comment]
Thanks @mamedin!

I'm not sure the best way to link to the SS application-specific environmental variables, other than what you've linked to and just hope we remember to update to the latest release branch upon each release. @sallain any recommendations?

**Comment by ablwr** [comment]
Updated! Ready for re-review @mamedin -- basically just took exactly what you said and added it in.

**Comment by sallain** [comment]
@ablwr I don't have a good answer for pointing to the right version in GitHub - I don't think there's a way to have GH auto-redirect users to the default branch. Maybe a note that reminds users to check that they're looking at the right branch in GitHub?

**Comment by ablwr** [comment]
Yeah, there's not. We could point it to qa/0.x but I'm not sure if that is the best thing to do either.

**Comment by sallain** [comment]
Unfortunately I think the answer is probably something tedious like creating a generic update to-do list for whenever we cut a new docs branch - one of the items would be "look for references to 1.10/0.15".

**Comment by ablwr** [comment]
OK if you're fine with how it is, it's ready for re-review!

**Review by sallain — APPROVED** [review]

**Comment by sallain** [comment]
@mamedin may have to re-review!

**Comment by ablwr** [comment]
I'll wait even though it's his words I cp-pasted :joy:

File: admin-manual/security/security.rst (hunk 1 above)
Comment (ablwr) [reply admin-manual/security/security.rst]: Please re-review!

**Review by mamedin — APPROVED** [review]
LGTM!

**Pull request merged**